wiki/en/en-Software-Manual.md

```diff
@@ -117,7 +117,7 @@ The faders allow you to adjust the level of what you hear without affecting what
 
 The VU meter shows the input level at the server - that is, the sound being sent.
 
-If you have set your Audio Channel to Stereo or Stereo Out in your Settings, you will also see a pan control (shift-click to reset).
+If you have set your Audio Channel to Stereo or Stereo Out in your Settings, you will also see a pan control (shift-right-click to reset).
```

Comment:
It's shift click for me on debian

Reply:
Yes, you have to be careful with the term "shift-click" because it can be different on different machines. I think in most docs it's hoped that the reader will know that it's shorthand for "shift-click the thing you normally do". I would leave it as  "shift-click" for that reason.

Reply:
@trebmuh could you revert this please?

Reply:
done